plugins/wallabag.koplugin/main.lua

```diff
@@ -397,30 +503,37 @@ The 'Synchronize remotely deleted files' option will remove local files that no
                 keep_menu_open = true,
                 callback = function()
                     UIManager:show(InfoMessage:new{
-                        text = T(_([[Wallabag is an open source read-it-later service. This plugin synchronizes with a Wallabag server.
+                        text = T(_([[wallabag is an open source read-it-later service. This plugin synchronizes with a wallabag server.
 
 More details: https://wallabag.org
 
-Downloads to folder: %1]]), BD.dirpath(filemanagerutil.abbreviate(self.directory)))
+Downloads to directory: %1]]), BD.dirpath(filemanagerutil.abbreviate(self.directory)))
                     })
                 end,
             },
         },
     }
-end
+end -- Wallabag:addToMainMenu
 
+--- Validate server settings and request an OAuth bearer token.
+-- Do not request a new token if the saved one is valid for more than 5 minutes.
 function Wallabag:getBearerToken()
+    local log_dbg = function(...) return log_dbg("getBearerToken:", ...) end
```

Comment:
It's a form of the same problem that exists with the very concept itself. Pleas revert this change because it has a horrible effect on greppability and clarity.


To illustrate, here's a random example of why we like it the way it is:
```
$ rg "Request failed"
frontend/apps/cloudstorage/webdavapi.lua
92:        logger.dbg("WebDavApi:listFolder: Request failed:", status or code)

frontend/apps/reader/modules/readerdictionary.lua
1148:            logger.dbg("ReaderDictionary: Request failed; response headers:", headers)
1173:        logger.dbg("ReaderDictionary: Request failed:", status or code)

plugins/wallabag.koplugin/main.lua
726:        logger.dbg("Wallabag: Request failed:", status or code)

plugins/opds.koplugin/opdsbrowser.lua
769:                logger.dbg("OPDSBrowser:downloadFile: Request failed:", status or code)

plugins/opds.koplugin/opdspse.lua
83:            logger.dbg("OPDSPSE:getLastPage: Progress Request failed:", progress_status or progress_code)
87:        logger.dbg("OPDSPSE:getLastPage: Authentication Request failed:", auth_status or auth_code)
144:                logger.dbg("OPDSBrowser:streamPages: Request failed:", status or code)
```

Reply:
Also not fond of this change, for another reason: even if our logging system doesn't log the class/object the message comes from, don't implement your own workaround. Stick with how things are done, for consistency and clarity.
We are fine without it, even if it makes things harder to debug, it keep things short. If one day we want it, we'll implement it in logger.lua with some introspection Lua trick.

Reply:
OK, I've removed these local log functions and reverted to the original `logger.dbg`, `.info`, etc.